### DISCUSSION TITLE
Elasticsearch shuts down

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#8699] [Unsupported Versions] [answered]
Hi, I am having problems with Elasticsearch. These are the specs for my setup

Standalone setup
16 gb ram
0 swap memory
Intel Core i3-8100 cpu
4 core cpu
5 tb SSD

In grafana the cpu is maxing out to 100 percent at time but the median is 55. The ram is median of 90, but the traffic I am seeing is in the kbs and go as high as 20 mbs at times. I think upgrading the ram to 32 will be the fix, but I am not entirely sure. Could this possibly be due to the upgrade to the new Elastic?

**Comment by reyesj2** [comment]
If you are just testing SO try running the EVAL mode for slightly better performance on your hardware specs. [Ref](https://docs.securityonion.net/en/2.3/architecture.html#evaluation)

If you are meaning this for a production environment you will need better specs for reliability. [Ref](https://docs.securityonion.net/en/2.3/hardware.html?highlight=hardware#standalone-deployments)

**Reply by its-a-me-mario-the-og** [reply]
It is for production environment, but before doing an upgrade on the specs. I would like to know if the specs for the new release is going to go up and by how much? So I can plan and get the upgrades made so it will work better for the new release.

**Reply by its-a-me-mario-the-og** [reply]
Thank you @reyesj2 for your help!

**Comment by dougburks** [comment, marked as answer]
> Standalone setup
16 gb ram
0 swap memory

From https://docs.securityonion.net/en/2.3/hardware.html#standalone-deployments:
_In a standalone deployment, the manager components and the sensor components all run on a single box, therefore, your hardware requirements will reflect that. You’ll need at minimum 16GB RAM, 4 CPU cores, and 200GB storage. At the bare minimum of 16GB RAM, you would most likely need swap space to avoid issues._

> I think upgrading the ram to 32 will be the fix, but I am not entirely sure.

If you can upgrade to 32GB RAM, then we'd highly recommend that.

**Reply by its-a-me-mario-the-og** [reply]
Thank you Doug for your help! I'll make the upgrade to 32GB of RAM soon.
